src/main/java/de/rabea/Router.java

```diff
@@ -0,0 +1,22 @@
+package de.rabea;
+
+import de.rabea.controller.NotFoundController;
+import de.rabea.request.HttpRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Router {
+    private Map<String, Controller> configuredRoutes = new HashMap<>();
+
+    public Controller getController(HttpRequest request) {
+        if (configuredRoutes.containsKey(request.requestLine().uri())) {
+            return configuredRoutes.get(request.requestLine().uri());
+        }
+        return new NotFoundController();
```

Comment:
No, I changed it. Although I didn't think that it mattered much to create a new instance every time? Is this for performance reasons?